app/src/main/java/io/intrepid/skeleton/screens/example2/Example2Contract.java

```diff
@@ -0,0 +1,15 @@
+package io.intrepid.skeleton.screens.example2;
+
+import io.intrepid.skeleton.base.BasePresenter;
+import io.intrepid.skeleton.base.BaseView;
+
+public class Example2Contract {
+    public interface View extends BaseView {
+
```

Comment:
Do we prefer View or Screen to denote the activity/fragment's "view" callback. I've started referring to classes that have a context as "Screen" since View is a very known android framework class. Housing these interfaces in a parent contract class alleviates the compilation error, maybe not so much the understanding of semantics.